x/stakeibc/types/params.go

```diff
@@ -26,6 +26,7 @@ var (
 	DefaultSafetyMaxRedemptionRateThreshold uint64 = 150           // divide by 100, so 150 = 1.5
 	DefaultMaxStakeICACallsPerEpoch         uint64 = 100
 	DefaultIBCTransferTimeoutNanos          uint64 = 1800000000000 // 30 minutes
+	DefaultSafetyNumValidators              uint64 = 30
```

Comment:
iirc we're currently at 35 for the Hub!

Reply:
I count 32. Good catch. Setting this to 35